### ISSUE TITLE
FBetaMultiLabelMeasure confusing docstring

### ISSUE THREAD

**Issue by david-waterworth** [allenai/allennlp#5472]
The docstring for the `threshold` parameter for `FBetaMultiLabelMeasure` states

    threshold: `float`, optional (default = `0.5`)
        Logits over this threshold will be considered predictions for the corresponding class.

This is slightly confusing, usually, you'd test if logits are greater than 0.0 or probabilities are greater than 0.5

The docstring implies that we should pass `logits` as `predictions`, but the default for threshold implies `probabilities` are expected.
